e2e/cypress/support/api/system.js

```diff
@@ -85,7 +85,7 @@ Cypress.Commands.add('apiUpdateConfig', (newConfig = {}) => {
             body: config,
         }).then((updateResponse) => {
             expect(updateResponse.status).to.equal(200);
-            return cy.wrap({config: response.body});
+            return cy.apiGetConfig();
```

Comment:
Hmm, weird. On update, I can see until [this line on server](https://github.com/mattermost/mattermost-server/blob/master/api4/config.go#L159) that it has been updated.